lib/tpm2_openssl.c

```diff
@@ -1018,6 +1018,37 @@ static bool load_public_AES_from_file(FILE *f, const char *path,
     return tpm2_util_calc_unique(name_alg, key, seed, unique);
 }
 
+static bool load_public_KEYEDHASH_from_file(FILE *f, const char *path,
+        TPM2B_PUBLIC *pub, TPM2B_SENSITIVE *priv) {
+
+    /*
+     * Get the file size and validate that it is within the allowed size for keyed hash objects
+     */
+    unsigned long file_size = 0;
+    bool result = files_get_file_size(f, &file_size, path);
+    if (!result) {
+        return false;
+    }
+
+    if (file_size > 128 || file_size == 0) {
```

Comment:
is the magic number 128 a TSS defined max value? use that instead.

Reply:
I couldn't find any define when looking in tss2_tpm2_types.h

Reply:
I thought it was MAX_SYM_DATA. However, In the 11.1.3 section of the part II of the specifications it allows for a max value of 128 but in tss2_tpm2_types.h it is 256. @AndreasFuchsSIT, @williamcroberts  do you know why the difference?

Reply:
@whooo can you macro define a label for the number 128?

Reply:
Sure, so something like ```#define KEYEDHASH_MAX_SIZE 128``` in lib/tpm2_openssl.c ?

Reply:
@idesai it's now added